packages/code-du-travail-frontend/pages/_document.js

```diff
@@ -49,7 +49,7 @@ export default class MyDocument extends Document {
             content="width=device-width, initial-scale=1, shrink-to-fit=no"
           />
           <link rel="shortcut icon" href="/static/favicon.ico" />
-          <script src="//polyfill.incubateur.social.gouv.fr/v3/polyfill.min.js?features=default%2CArray.prototype.includes%2CArray.prototype.find%2CArray.prototype.findIndex%2CObject.setPrototypeOf%2CObject.values%2CNumber.isFinite%2CSymbol%2CSymbol.hasInstance%2CSymbol.isConcatSpreadable%2CSymbol.iterator%2CSymbol.unscopables%2CSymbol.toStringTag%2CSymbol.toPrimitive%2CSymbol.split%2CSymbol.search%2CSymbol.species%2CSymbol.replace%2CSymbol.match%2CSet%2CMap%2CWeakMap" />
+          <script src="https://polyfill.io/v3/polyfill.min.js?features=es6%2Ces2016%2Ces2017" />
```

Comment:
we shouldn't use external polyfill as it may add some tracking. also, another PR  maybe ?

Reply:
oupsy